### ISSUE TITLE
Can't open .flatpakref files with Warehouse

### ISSUE THREAD

**Issue by Xrey274** [flattool/warehouse#76]
Currently using Solus Budgie, everything up to date. Inside of nautilus you can open files with the "Open With." option, where you can choose what program to run/open the file with. Sadly Warehouse doesn't appear on this list, i think it's not creating some kind of references that you can be used to open flatpakref files directly.

**Comment by heliguy4599** [comment]
That's correct, I do not currently have anything in Warehouse that would make this possible. It's a feature I have in mind to add but am not entirely sure how to just yet. Thanks.

**Comment by bleonard252** [comment]
Warehouse now offers an "open file" option in its menu. It should be trivial to reuse that for this, right?

(Also, tangentially, how hard would it be to accept Appstream URLs, i.e. from the Flatline browser extension, and look them up in the Install Packages page?)

**Comment by brunos3d** [comment]
Hello, folks.
Here is my PR helping to implement this feature, hope it helps. 

https://github.com/flattool/warehouse/pull/177

cc: @heliguy4599 @bleonard252 @Xrey274
